test/unit/Search.test.js

```diff
@@ -198,5 +198,18 @@ describe('Search', () => {
       component.find('button').at(0).simulate('click');
       expect(store.state.searchKeywords).not.to.equal('Watts');
     });
+
+    it('should make a search request with issuance filter set for journal title searches', () => {
+      component.find('select').getDOMNode().value = 'journal title';
+      component.find('select').simulate('change');
+
+      mock = new MockAdapter(axios);
+      mock
```

Comment:
What about a test to check that this call was made?

Reply:
how would I do that? I couldn't find an option in that module to check how it was invoked outside of looking at the response. I may have missed something though, it's been a while since I've written react tests.

Reply:
I thought the axios mock adapter had something to test that, but having trouble finding it. I'm just not seeing how this test makes sure that the right api url is used.

You could create a spy for the push method, I believe, and check the argument value.

Reply:
So, I didn't create a spy, because the `push` method is actually just stubbed out as an anonymous function, but I just updated that so it pushes the route to an array where we can verify it